### DISCUSSION TITLE
Understanding the "Scheduled background jobs: 0"

### DISCUSSION THREAD

**Discussion by johnhelt** [nextcloud/recognize#1040] [Q&A]
Hi. I'm running recognize on my docker installation of nextcloud using docker-compose. The compose file also launches a dedicated cron image that runs the same nextcloud image as the "app", as suggested by the docker-compose yaml in nextcloud docker (https://github.com/nextcloud/docker/blob/master/.examples/docker-compose/with-nginx-proxy/mariadb/fpm/docker-compose.yml).

My understanding was the this cronjob takes care of launching all scheduled jobs. I couldn't find anywhere in the recognize settings to set these background jobs up, so I was under the impression this takes place under the hood. Now, I'm not seeing any background jobs scheduled for recognize after I installed it. In fact, I have the following message shown in the admin settings for recognize:

```
Face recognition: 0 Queued files, Last classification: 15 hours ago, Scheduled background jobs: 0,
Face clustering: 41 faces left to cluster, Last clustering run: 10 hours ago, Scheduled background jobs: 0,
A minimum of 120 faces per user is necessary for clustering to kick in
```
FYI I have added several more images to classify and cluster since I ran the initial detection. I did the initial setup by running the commands from my "app" container, because I wanted to do "factory reset":

occ recognize:clear-background-jobs
occ recognize:reset-face-clusters
occ recognize:reset-faces
occ recognize:classify
occ recognize:cluster-faces

Thanks for the app btw, I'm very happy to use it!

**Comment by marcelklehr** [comment]
> Face recognition: 0 Queued files, Last classification: 15 hours ago, Scheduled background jobs: 0,

This is a good sign. 0 queued files means everything that should have been processed was processed. 0 Scheduled background jobs means, nothing is currently scheduled to be done. Also good, since there is nothing queued to be processed.

> 41 faces left to cluster

This is a bit unfortunate as it seems you have too few face photos for clustering to kick in -- unless you have a lot of clustered faces already?

**Comment by johnhelt** [comment]
Hi! Thanks for your reply. Unfortunately, it seems that any pictures I added since I installed recognize have not yet been processed. I can see that, since a) they're not showing up in the existing people categories and b) they're not showing up as "unassigned faces". How could that be?

**Reply by johnhelt** [reply]
I just wanted to give an update. After disabling the app, and re-enabling the app from the APPS panel in nextcloud, I can now see changes  in the recognize admin view:

```
Face recognition: 3820 Queued files, Last classification: 2 days ago, Scheduled background jobs: 2,
Face clustering: 3 faces left to cluster, Last clustering run: 10 hours ago, Scheduled background jobs: 0,
A minimum of 120 faces per user is necessary for clustering to kick in
```

So apparently, something was stuck, and got unstuck with this.

**Reply by johnhelt** [reply]
Another update: I still don't have new faces showing up in either categories (people, and unassigned). I keep getting a "face score too low" in the nextcloud.log. On top of this, there is about 7000 images waiting to be clustered, but that job hasn't run since 3rd of December (why?).

I also commented this in an issue I found (which was closed because it went stale, however). So I'm no where near a nice functional setup, unfortunately.
